docs/source/federated-types/composition.mdx

```diff
@@ -248,3 +248,254 @@ type User @key(fields: "id") {
 If you make a shared type an [entity](../entities/), different subgraphs can define any number of different fields for that type, as long as they all define **key fields** for it.
 
 This is a useful solution when a type corresponds closely to an entry in a data store that one or more of your subgraphs has access to (e.g., a `Users` database).
+
+## Merging types from multiple subgraphs
+
+If a particular GraphQL type is defined _differently_ by different subgraphs, composition uses one of two strategies to merge those definitions: **union** or **intersection**.
+
+* **Union**: The supergraph schema includes _all_ parts of _all_ subgraph definitions for the type. 
+* **Intersection:** The supergraph schema includes _only_ the parts of the type that are present in _every_ subgraph that defines the type.
+
+The merging strategy that composition uses for a particular type depends on the type, as described below.
+
+### Object, union, and interface types
+
+Composition _always_ uses the union strategy to merge object, union, and interface types.
+
+Consider the following subgraph schemas:
+
+<CodeColumns>
+
+```graphql title="Subgraph A"
+type User @key(fields: "id") {
+  id: ID!
+  name: String!
+  email: String!
+}
+
+union Media = Book | Movie
+
+interface BookDetails {
+  title: String!
+  author: String!
+}
+```
+
+```graphql title="Subgraph B"
+type User @key(fields: "id") {
+  id: ID!
+  age: Int!
+}
+
+union Media = Book | Podcast
+
+interface BookDetails {
+  title: String!
+  numPages: Int
+}
+```
+
+<></>
+
+</CodeColumns>
+
+When these subgraph schemas are composed, the composition process merges the three corresponding types using the union strategy. This results in the following type definitions in the supergraph schema:
```

Comment:
General comment that occurs in multiple places. This may just be my personal taste, but I find "using the union strategy" to be a little clunky. I would personally go with "merge by union" or "by union". Probably a matter of taste, so feel free to ignore.